Makefile

```diff
@@ -44,10 +44,9 @@ ifdef SHOULD_LINT
 	@$(foreach dir,$(PKGS),golint $(dir) 2>&1 | tee -a lint.log;)
 	@echo "Checking for unresolved FIXMEs..."
 	@git grep -i fixme | grep -v -e vendor -e Makefile | tee -a lint.log
-	@echo "Ensuring generated doc.go are up to date"
-	$(ECHO_V)$(MAKE) gendoc
 	@echo "Checking for license headers..."
-	@./check_license.sh | tee -a lint.log
+	@DRY_RUN=1 ./check_license.sh | tee -a lint.log
+	@$(MAKE) gendoc
```

Comment:
This isn't a major concern, TBH. One way is obviously to check if the git working tree is clean. Another would be to make md-to-godoc support a `--dry-run`/`--diff` flag which reports what changes would be made rather than actually changing.